.eslintignore

```diff
@@ -0,0 +1,2 @@
+dist
+example
```

Comment:
For now just disabled this and removed the react rules, we can bring it back later if we want but didn't seem as important and was causing some issues on my machine.